src/ansys/units/quantity_tables/cgs_table.yaml

```diff
@@ -8,91 +8,85 @@ quantity_units_table:
   Length: cm
   Time: s
   Temperature: K
-  Angle: radian
-  Solid Angle: sr
-  Chemical Amount: mol
-  Energy: erg
-  Force: dyne
-  Power: erg s^-1
-  Pressure: dyne cm^-2
   Current: A
-  Voltage: V
-  Luminance: cd
+  SubstanceAmount: mol
+  Light: cd
+  Angle: radian
+  SolidAngle: sr
   Acceleration: cm s^-2
-  Angular Acceleration: radian s^-2
-  Angular Velocity: radian s^-1
+  AngularAcceleration: radian s^-2
+  AngularVelocity: radian s^-1
   Area: cm^2
+  Capacitance: farad
```

Comment:
![image](https://github.com/ansys/pyansys-units/assets/106588300/dec5d238-a8ba-415c-bef8-81075c53222c)

I think it is incorrect because farad is defined with SI units.

Reply:
@seanpearsonuk 

  N:
    composition: kg m s^-2
    factor: 1  
V:
    composition: A ohm
    factor: 1
  farad:
    composition: N m V^-2
    factor: 1

Reply:
We have removed all incorrect units including this.